CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/panes/Bundle.properties

```diff
@@ -1,8 +1,6 @@
 DataAccessViewCategoryPanel.jLabel3.text=Select Data Access View categories
-DataAccessViewCategoryPanel.buttonSingleRight.text=>
-DataAccessViewCategoryPanel.buttonSingleLeft.text=<
 DataAccessViewCategoryPanel.jLabel1.text=Visible categories
 DataAccessViewCategoryPanel.jLabel2.text=Hidden categories
 DataAccessViewCategoryPanel.jLabel4.text=Changes take effect after restarting Constellation.
-DataAccessViewCategoryPanel.buttonDoubleRight.text=>>
-DataAccessViewCategoryPanel.buttonDoubleLeft.text=<<
+DataAccessViewCategoryPanel.buttonDoubleRight.text=>
+DataAccessViewCategoryPanel.buttonDoubleLeft.text=<
```

Comment:
Don't know if it might be worthwhile renaming buttonDoubleRight/Left to simply buttonRight/Left? Given an arrow has been omitted from this option (so it's not really double anymore) and the other option was removed entirely (so it's now the only option)

Reply:
ok. that make sense.